exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,12 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.includes("code")) {
```

Comment:
Great use of "return early" which is often referred to as a best practice, by removing the "else" you've simplified your code, in a very large function the benefit of this practice would be more obvious. Good job.

You could actually use this practice in some of your previous exercise submissions in the PR but it isn't required.